### DISCUSSION TITLE
How to delete a file everywhere/in the cloud/remotely?

### DISCUSSION THREAD

**Discussion by zuckerruebe** [iterate-ch/cyberduck#14986] [Q&A] [answered]
Hi all

How can I delete a file in the cloud (in my case Wasabi) through Mountain Duck? When I delete the file in the mounted directory in Windows explorer all that happens is that the file's icon changes from the green tick mark to a small cloud:

Before:

![grafik](https://github.com/iterate-ch/cyberduck/assets/11159050/6c7e47f7-f918-437b-ac5c-4a5ee7c79a9d)

After:

![grafik](https://github.com/iterate-ch/cyberduck/assets/11159050/c8f60eda-e0ee-45d6-8b4d-da4d30e94f72)

However, I don't want to delete the file just locally but in the cloud and anywhere that synchronizes to the cloud.

How can I accomplish this? Do I have to go through the Wasabi web interface? But that won't work with Cryptomator, will it? Or maybe it's a user rights issue?

Cheers,
Damian

**Comment by dkocher** [comment, marked as answer]
This indicates the delete operation fails with Wasabi and thus the file is still found on the server although no longer cached. Please check for permission issues deleting files in your account.

**Reply by zuckerruebe** [reply]
Thanks. I had to assign the `WasabiFullAccess` policy to my user. Apparently, `WasabiWriteOnlyAccess` and `WasabiReadOnlyAccess` is not enough to delete files.
